build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml

```diff
@@ -54,3 +55,14 @@ jobs:
     inputs:
       PathtoPublish: '${{ parameters.nupkgdir }}'
       artifactName: 'drop'
+
+  - ${{if parameters.publishVstsFeed }}:
+    - template: MUX-InstallNuget-Steps.yml
```

Comment:
> template: MUX-InstallNuget-Steps.yml [](start = 5, length = 37)

Isn't this already done above?

Reply:
Oh, yeah. Initially I was authoring it as a different job. Will remove.